src/java.base/share/man/keytool.1

```diff
@@ -453,15 +453,15 @@ specified.
 The \f[CB]\-signer\f[R] value specifies the alias of a
 \f[CB]PrivateKeyEntry\f[R] for the signer that already exists in the
 keystore.
-This option is used to sign the certificate with the signer???s private
+This option is used to sign the certificate with the signer?s private
```

Comment:
Not a problem with this PR as such, but we still have a `?` character in the output.

Reply:
Yeah I spotted that too, but it would need to be fixed in source and nroff.  Must be some kind of "smart quote" from an editor. Do you think this needs to be fixed or just handle it in mainline?

Reply:
Filed [JDK-8290626](https://bugs.openjdk.org/browse/JDK-8290626). It can easily be fixed before RDP2.

Reply:
Hello @jonathan-gibbons

Excuse my hijacking / piggy-backing on this conversation!

When you say

> Not a problem with this PR as such, but we still have a `?` character in the output.

This strongly suggests that there must be an input to those man pages somewhere. :-) Would you mind to point me to it? Are they even included in the git repo?

Also, I was wondering why these man pages (i.e. the outputs) are obviously not included any more in the OpenJDK builds, e.g. https://jdk.java.net/19/ ? I am not sure when they were removed or if they were ever included in _open_jdk builds at all, but I see them in the Oracle JDK 1.8 Build.

I find them really nice to read! Are there other output formats, e.g. HTML, that might be are deployed somewhere on the web?

Reply:
It did not take much effort to answer the last of the above question myself.

> Are there other output formats, e.g. HTML, that might be are deployed somewhere on the web?

Obviously "yes". HTML renderings can be found by following the, e.g. "[Java Development Kit Version 18 Tool Specifications](https://docs.oracle.com/en/java/javase/18/docs/specs/man/index.html)" Link on the "Specifications Overview" page at, e.g. "https://docs.oracle.com/en/java/javase/18/docs/specs/".

Still hunting for the sources.